libs/wingsdk/src/shared-aws/function.inflight.ts

```diff
@@ -22,7 +22,10 @@ export class FunctionClient implements IFunctionClient {
   public async invoke(payload?: string): Promise<string | undefined> {
     const command = new InvokeCommand({
       FunctionName: this.functionArn,
-      Payload: payload ? fromUtf8(JSON.stringify(payload)) : undefined,
+      // If payload is undefined, pass json `null` as the payload to the function
+      // to ensure the received event will be `null` (which will be converted to `undefined` in the function code)
+      // If the Payload is undefined, the resulting event will instead be `{}`
+      Payload: fromUtf8(payload ? JSON.stringify(payload) : "null"),
```

Comment:
what if the payload is empty string or 0? `payload ? JSON.stringify(payload) : "null"` would evaluate to "null" instead of "\"\"" or "0" respectively

Reply:
0 wouldn't be relevent since this value is either a string or undefined, but you're right about empty string (plus being explicit is good regardless)